docs/rollup.config.js

```diff
@@ -12,9 +12,21 @@ module.exports = async () => {
     // development mode creates a non-minified build for debugging or development
     developmentMode: false, // process.env.ROLLUP_WATCH === 'true',
 
-    // set to true to inject the service worker registration into your index.html
     injectServiceWorker: false,
-    workbox: false,
+    workbox: {
```

Comment:
This config don't override the default, just merge with it, right?

Reply:
oo, good catch, thanks!